snap/snapcraft.yaml

```diff
@@ -54,11 +52,7 @@ parts:
       craftctl set version=${VERSION}
 
 plugs:
-  dot-local-share-juju:
```

Comment:
I think that this is needed, because we are accessing it with lubjuju, or not?

Reply:
Simple reading access would be enough, but we need to be able to read from there.

Reply:
The [juju-client-observe-interface](https://snapcraft.io/docs/juju-client-observe-interface) already give us read access to .local/share/juju used in libjuju

Reply:
Perfect 👍